src/main/java/tc/oc/pgm/modules/ModifyBowProjectileMatchModule.java

```diff
@@ -25,13 +21,13 @@
 
 @ListenerScope(MatchScope.RUNNING)
 public class ModifyBowProjectileMatchModule extends MatchModule implements Listener {
-  protected final Class<? extends Entity> cls;
+  protected final Class<? extends Projectile> cls;
```

Comment:
PGM allows for entities to be shot, not necessarily just projectiles, I don't think that functionality should be removed.

Reply:
If this is something we're interested in keeping I can address it in the next couple of days, my thought process here was to simplify the solution using a native method that we know works. A lot of these custom entities have problems, for example: deaths caused by explosions, splash potion events, etc. I left the projectile definitions as is (entities) so this limit is just for this specific module.

Reply:
Although entities were technically supported, it was usually used to do weird stuff like creeper bows -- which caused other sorts of bugs.

To fix the ender pearl issue, I'm okay merging this fix.